doc/sphinx/explore-mir-using-demos.md

```diff
@@ -0,0 +1,134 @@
+# Explore What Mir Is Capable of Using Mir Demos
+Mir comes with a set of demos in its [example
+directory](https://github.com/canonical/mir/tree/main/examples) to showcase its
+abilities. This section will cover the installation and usage of these demos.
+
+
+## Installation
+Mir demos are available on Debian derivatives, Fedora, and Alpine. For distros
+that don't have prebuilt binaries, examples can be built from source.
+
+<details>
+<summary> Installing Mir Demos on Debian and its derivatives </summary>
+
+```sh
+sudo apt install mir-demos mir-graphics-drivers-desktop
+```
+</details>
+
+<details>
+<summary> Installing Mir Demos on Fedora </summary>
+
+```sh
+sudo dnf install mir-demos mir
```

Comment:
I would expect mir to be a dependency of mir-demos do we know both need to be installed?
```suggestion
sudo dnf install mir-demos
```